javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var exceptNull = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      values.splice(i, 1, i + 1);
```

Comment:
Oh noooooooo, this is bad news because you are changing the original array. You want to use the values of the array given as an argument, but you never want to change the original values.

Reply:
ditched splice